### PR TITLE
Different files

### PR DESCRIPTION
**Pull request by Fyurij**

### CODE REVIEW

File: Figures.h [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#pragma once
+#include <string>
+class Figure
+{
+public:
+    virtual int GetX() = 0;
+    virtual int GetY() = 0;
+    virtual std::string GetName() = 0;
```
Comment (keyrnk) [inline Figures.h]: it's better to return std::string&

File: Figures.h [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+};
+
+class King : public Figure
+{
+private:
+    int x;
+    int y;
+    std::string name;
+public:
+    King(int x_, int y_, std::string name_)
```
Comment (keyrnk) [inline Figures.h]: it's better to pass name_ by const std::string&

File: Figures.cpp [hunk 3]
```diff
@@ -0,0 +1,109 @@
+#include <iostream>
```
Comment (keyrnk) [inline Figures.cpp:1]: I think it's better to split .cpp by separate files

File: Figures.cpp [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+{
+    return y;
+}
+std::string King::GetName()
+{
+    return name;
+}
+bool King::Danger(int i, int j)
+{
+    if (abs(x - i) <= 1 && abs(y - j) <= 1)
```
Comment (keyrnk) [inline Figures.cpp]: change to return in one string

File: Figures.cpp [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+{
+    return y;
+}
+std::string Queen::GetName()
+{
+    return name;
+}
+bool Queen::Danger(int i, int j)
+{
+    if (x == i || y == j || (x - y) == (i - j))
```
Comment (keyrnk) [inline Figures.cpp]: the same return in one string

File: Getting_Ready_to_Chess.cpp [hunk 6]
```diff
@@ -199,7 +21,7 @@ int main()
     while (!koordinates.eof())
     {
```
Comment (keyrnk) [inline Getting_Ready_to_Chess.cpp:20]: please put all code for reading from file to a separate function, e. g. ReadFiguresFromFile()

File: Figures.cpp [hunk 7]
```diff
@@ -0,0 +1,109 @@
+#include <iostream>
+#include <string>
+#include "Figures.h"
```
Comment (Fyurij) [inline Figures.cpp:3]: qwerty

**Review by keyrnk — APPROVED** [review]

**Pull request merged**